test/cpp/run_tests.sh

```diff
@@ -9,6 +9,7 @@ BUILD_ONLY=0
 RMBUILD=1
 LOGFILE=/tmp/pytorch_cpp_test.log
 XLA_EXPERIMENTAL="nonzero:masked_select"
+export XLA_SYNC_BEFORE_ITEM_CALL=0
```

Comment:
I think we can leave this file alone and add
```
./run_tests.sh -X early_sync -F AtenXlaTensorTest.TestEarlySyncLiveTensors
```
after
https://github.com/pytorch/xla/blob/ea15839301cfbc10841ef76089f1f1ebaaed538e/.circleci/test.sh#L29

The reason being that `run_tests.sh` can be used to run a single test and we don't want to run `TestEarlySyncLiveTensors` if that is the case. This should get your test run with our CI on cpu and gpu.